### ISSUE TITLE
Option to disable routing

### ISSUE THREAD

**Issue by stephanebruckert** [dbbk/ember-routable-modal#6]
This might sound like a weird suggestion, but it would be nice to have an option to make a modal non-routable (basically being able to disable the primary feature of this addon.

Here is why:

 1. in the case of the security modal, i.e. "Please confirm you really want to delete your account", we _should not_ be able to access that route directly: example.com/user/settings/delete-confirmation
 2. for styling and to avoid code duplication, to fix **1.** we don't want to move to another, second ember modal addon such as https://github.com/yapplabs/ember-modal-dialog

What do you think?

**Comment by dbbk** [comment]
Agreed, I'd also like to consolidate on using just one modal addon rather than two! Development has been a little slow on this addon recently but I'm going to be carving out some time soon to focus on it and this is definitely one of the things I want to implement 👍
